web/src/utils/constants.ts

```diff
@@ -1,7 +1,7 @@
 import type { Duration } from 'date-fns';
 import { ElectricityModeType } from 'types';
 
-export const metaTitleSuffix = ' | App | Electricity Maps';
```

Comment:
Do we really want to change this?

It currently follows the same style as our other websites and describes what they are rather than what they do.

Blog, Docs, etc

Reply:
Whoops, didn't mean to include this change in this PR.